docs/examples/dataclasses_default_schema.py

```diff
@@ -0,0 +1,12 @@
+import dataclasses
+from typing import List
+from pydantic.dataclasses import dataclass
+
+@dataclass
+class User:
+    id: int
+    name: str = 'John Doe'
+    friends: List[int] = dataclasses.field(default_factory=lambda: [0])
+
+user = User(id='42')
+print(user.__pydantic_model__.schema()['properties']['friends'])
```

Comment:
```suggestion
print(user.__pydantic_model__.schema())
```
Is probably enough, best to keep documentation as simple as possible while demonstrating the point.

Reply:
I had this at first then thought the prints use up too much space. So you prefer the complete schema for sure?

Reply:
yes please.